client/components/Editor/Slides.css

```diff
@@ -12,3 +12,18 @@
 .Slides-slide-sidebar-container {
     margin: 1em 0;
 }
+
+.Slides-slide-sidebar-container:first-of-type {
+    margin: 3px 0 0 0;
+}
+
+.Slides-list-container {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 65vh;
+    max-height: 100%;
+    padding: 1px !important;
+    border: 0 !important;
+    box-shadow: 0 0 0 0 !important;
+    -webkit-box-shadow: 0 0 0 0 !important;
```

Comment:
This is a lot of `important` tags. Not that I'm necessarily opposed, but what styling are we fighting against here? With Semantic UI sometimes I've had trouble targeting elements. In those cases, I've also used some of their existing Semantic UI classes for targeting. I don't know if you think that's better or worse than `important` tags. Open to suggestions!

Example CSS: https://github.com/bocoup/threeflows/blob/teacher-moments/client/components/SlideComponentsList/SlideComponentsList.css#L1